docs/ci-integration/remote-buildkit.md

```diff
@@ -0,0 +1,63 @@
+# Remote Buildkit
+
+## Introduction
+
+In some cases, you may want to run a remote instance of `earthly/buildkitd`. This guide is intended to help you identify if you might benefit from this configuration, and to help you set it up correctly.
+
+### Why Remote?
+
+Running a remote daemon is a unique feature of Earthly. It allows the build to happen elsewhere; even when executing it from your local development machine. However, it is not always the best  option. Before setting up a remote daemon, first look into Earthly's shared caching capabilities and see if those can get you the boost you need. In our experience, shared caching is usually enough.
+
+However, there are instances where a remote daemon can make the most sense. Here are some examples:
+
+- You have a single, powerful build machine you would like to share with your development team
+- There is data closer to a remote machine than your development/CI environment, so you bring the build to the data
+- You are using Earthly in Kubernetes, and want to isolate the containers doing the actual building because they require privileged mode
+- You want to share a build machine (or cluster) with your CI environment and your developers
+- Your local computer does not have the capabilities to build the software (`docker` is missing, or you lack sufficient privileges, or it is simply not powerful enough)
+
+### Configuring A Remote Cache
+
+#### Networking
+
+A remote daemon should be reachable by all clients intending to use it. Earthly uses ports `8371-8373` to communicate, so these should be open and available to communicate.
+
+#### Daemon
+
+To configure an `earhly/buildkitd` daemon as a remotely available daemon, you will need to start the container yourself. See our configuration docs for more details on all the options available; but here are the ones you need to know:
+
+**`BUILDKIT_TCP_TRANSPORT_ENABLED`**
+
+This will configure `buildkitd` on start to start listening on port `8372`. If you would like it to be externally available on a different port, you will need to handle that at the port mapping level. TCP is required for remotely sharing a daemon.
+
+**`BUILDKIT_TLS_ENABLED`**
+
+Set this to `true` for all daemons that will handle production workloads. This daemon *by design* is an arbitrary code execution machine, and running it without any kind of mTLS configuration is not recommended.
+
+Make sure you mount your your certificates and keys into the correct location (`/etc/*.pem`).
+
+#### Client
+
+Normally, Earthly will try to start and manage its own `earthly/buildkitd` daemon. However, with some configuration, it will not try to create or manage this daemon and rely on the remote one. Here are the configuration values you may need to know:
```

Comment:
```suggestion
Normally, Earthly will try to start and manage its own `earthly/buildkitd` daemon; however you can override this behaviour with the following configuration options:
```

Reply:
I don't like mine, yours is better, but I think I can get at what I meant with some rework here.

Reply:
sounds good, it was just a suggestion that you might be able to make it a bit more concise.

Reply:
I changed it. Thanks!